docs/tutorials/proposals/network-parameter-proposal.md

```diff
@@ -65,6 +65,8 @@ For example, a proposal to change the `rewards.activityStreak.benefitTiers` netw
 }"
 ```
```

Comment:
Similar to the other comment, this JSON could be made valid by wrapping the whole thing in `{...}`. It's not required, or important.